ldcbench.data-generator/src/main/java/org/dice_research/ldcbench/benchmark/DataGenerator.java

```diff
@@ -337,6 +337,12 @@ public void init() throws Exception {
             generator.generateGraph(avgDegree, numberOfEdges, seed, graph);
         }
 
+        if (type == Types.RDF_GRAPH_GENERATOR) {
+            //CHOOSE THE NUMBER OF BLANK NODE?? Currently using a fixed number of bNode
+            //DECIDE IF WE WANT BLANK NODES OR NOT ??
+            graph.addBlankNodes(3);
+        }
```

Comment:
Please use a user-defined parameter as discussed. This would also have to be added to the benchmark.ttl file.